packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx

```diff
@@ -270,18 +270,19 @@ export const SendTokenScreen: FC = () => {
   const maxFee = "3000000000000000"  // FIXME: hardcoded to 0.003 ALPH for now
 
   const setMaxInputAmount = useCallback(
-    (token: TokenWithBalance, maxFee?: string) => {
+    (token: TokenWithBalance) => {
       const tokenDecimals = token.decimals ?? 18
       const tokenBalance = formatTokenBalance(token.balance, tokenDecimals)
 
-      if (token.balance && maxFee) {
-        const balanceBn = token.balance
-
-        const maxAmount =
-          account?.networkId ===
-            "devnet" /** FIXME: workaround for localhost fee estimate with devnet 0.3.4 */
-            ? balanceBn.sub(maxFee).sub(100000000000000)
-            : balanceBn.sub(maxFee)
+      if (token.balance) {
+        let maxAmount = token.balance
+        if (feeToken?.id === token.id) {
+          maxAmount =
+            account?.networkId ===
+              "devnet" /** FIXME: workaround for localhost fee estimate with devnet 0.3.4 */
```

Comment:
What was the issue for devnet?

Reply:
Tested on devnet, doesn't have any issues, removing this now.